cpu/msp430_common/malloc.c

```diff
@@ -1,486 +0,0 @@
-/*
```

Comment:
Is `malloc` provided now by the newlibc?

Reply:
yup!